.changeset/slow-ligers-tie.md

```diff
@@ -0,0 +1,5 @@
+---
+'@capawesome/capacitor-posthog': major
```

Comment:
@robingenz is major correct? This was generated automatically. Additionally do we have to update the version in the package.json of the posthog package manually?